### PR TITLE
refactor: replace util.SliceContainsString with slices.Contains & make fmt

### PR DESCRIPTION
**Pull request by izturn**
## 📑 Description

replace util.SliceContainsString with  slices.Contains and run `make fmt`

## ✅ Checks

- [X] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [X] All the tests have passed

## ℹ Additional Information

Signed-off-by: Gang Liu [gang.liu@daocloud.io](mailto:gang.liu@daocloud.io)

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -207,7 +199,7 @@ func FileExists(path string) (bool, error) {
 }
 
 func EnsureDirExists(dir string) error {
-	err := os.MkdirAll(dir, 0755)
+	err := os.MkdirAll(dir, 0o755)
```
Comment (JuHyung-Son) [inline pkg/util/util.go:202]: why 0o755??

**Review by JuHyung-Son — COMMENTED** [review]
welcome! and thanks for clean code.
i left two questions

File: pkg/util/util.go [hunk 2]
```diff
@@ -183,7 +174,8 @@ func GetCacheKey(provider string, language string, sEnc string) string {
 
 func GetPodListByLabels(client k.Interface,
 	namespace string,
-	labels map[string]string) (*v1.PodList, error) {
+	labels map[string]string,
```
[the remark this answers is not in the document]
Comment (izturn) [reply pkg/util/util.go:177]: yes

File: pkg/util/util.go (hunk 1 above)
Comment (izturn) [reply pkg/util/util.go:202]: output of `make fmt`

Comment (AlexsJones) [reply pkg/util/util.go:202]: Really? 755 is a permission but 0o755, I dont know what that `0o` brings

Comment (arbreezy) [reply pkg/util/util.go:202]: Indeed this is an actual thing, it indicates an octal number so it's safe.

**Pull request merged**